docs/en/engines/table-engines/mergetree-family/invertedindexes.md

```diff
@@ -161,13 +161,12 @@ Advanced users can specify a different index granularity (we do not recommend th
 The default values of the following advanced parameters will work well in virtually all situations.
 We do not recommend changing them.
 
-Optional parameter `dictionary_block_size` (default: 128) specifies the size of dictionary blocks in rows.
+Optional parameter `dictionary_block_size` (default: 512) specifies the size of dictionary blocks in rows.
 
 Optional parameter `dictionary_block_frontcoding_compression` (default: 1) specifies if the dictionary blocks use front coding as compression.
 
-Optional parameter `max_cardinality_for_embedded_postings` (default: 16) specifies the cardinality threshold below which posting lists should be embedded into dictionary blocks.
+Optional parameter `posting_list_block_size` (default: 1048576) specifies the size of posting list blocks rows.
```

Comment:
We need to add `posting_list_block_size` in the CREATE TABLE statement at l. 47.